### ISSUE TITLE
Version v1.2.12 breaks in Chef 11.x

### ISSUE THREAD

**Issue by benholtz** [opscode-cookbooks/xml#23]
It looks like my issue may be related to #22  #21  Please fix ASAP! 

```
================================================================================
Recipe Compile Error in /var/lib/aws/opsworks/cache.stage2/cookbooks/dnsupdate/recipes/add.rb
================================================================================
 
 
ArgumentError
-------------
wrong number of arguments (1 for 0)
 
 
Cookbook Trace:
---------------
/var/lib/aws/opsworks/cache.stage2/cookbooks/xml/recipes/ruby.rb:44:in `block in from_file'
/var/lib/aws/opsworks/cache.stage2/cookbooks/xml/recipes/ruby.rb:43:in `from_file'
/var/lib/aws/opsworks/cache.stage2/cookbooks/route53/recipes/default.rb:20:in `from_file'
/var/lib/aws/opsworks/cache.stage2/cookbooks/dnsupdate/recipes/add.rb:2:in `from_file'
 
 
Relevant File Content:
----------------------
/var/lib/aws/opsworks/cache.stage2/cookbooks/xml/recipes/ruby.rb:
 
37:      Chef::Application.fatal!("You must specify a version less than or equal to 1.6.1 of nokogiri to use system libraries. You set: #{node['xml']['nokogiri']['version']}.")
38:    else
39:      ENV['NOKOGIRI_USE_SYSTEM_LIBRARIES'] = node['xml']['nokogiri']['use_system_libraries'].to_s
40:    end
41:  end
42:  
43:  chef_gem 'nokogiri' do
44>>   compile_time true if Chef::Resource::ChefGem.method_defined?(:compile_time)
45:    version node['xml']['nokogiri']['version']
46:    action :install
47:  end
48:

**Comment by scalp42** [comment]
Related to: https://github.com/opscode-cookbooks/aws/pull/110

**Comment by data-wiz** [comment]
any development on this?

**Comment by someara** [comment]
Should be fixed in 1.2.13



—

**Comment by benholtz** [comment]
Indeed it is fixed in 1.2.13
